.github/workflows/check-gradle.yml

```diff
@@ -0,0 +1,17 @@
+name: Gradle check
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - 'gradle/**'
```

Comment:
I pointed out in my [comment](https://github.com/Kotlin/dokka/pull/2951#discussion_r1153612825) that I intentionally want to run it for every commit in master in case anything is done to it indirectly, or the change is force pushed and the workflow result deleted, or the filenames/paths are changed without updating the paths here, and so on - it then has at least **some** chance of catching a bad actor.

Reply:
Indeed.